CONTRIBUTING.md

```diff
@@ -15,23 +15,23 @@ We use Codacy to check Pull Requests (PRs). PRs need to pass checks before they
 All (new) methods and functions have to be unittested using Python `doctests`.
 
 ### Reviewing
-All changes (not matter how minor) to master are made through Pull Requests (PRs). 
-All PRs are reviewed by at least one reviewer. 
+All changes (not matter how minor) to master are made through Pull Requests (PRs).
+All PRs are reviewed by at least one reviewer.
 If you know that someone may have an opinion on your change, make sure this person reviewed your code before merging.
 
 ### CI/CD
-We run a Continues Integration (CI) pipeline on [Travis](Travis). Travis continuesly runs `script/test`, which in turn runs all tests in the `tests` folder and all `doctests`.
+We run a Continues Integration (CI) pipeline on [Travis](Travis). Travis continuously runs `script/test`, which in turn runs all tests in the `tests` folder and all `doctests`.
```

Comment:
`Continues Integration` is still in here; should also be fixed. Also on line 26 (`Continues Deployment`).